Packs/VaronisSaaS/Integrations/VaronisSaaS/VaronisSaaS.py

```diff
@@ -0,0 +1,2108 @@
+            if last_days:
+                time_condition\
+                    .set_operator("LastDays")\
+                    .add_value({alert_attributes.Alert_TimeUTC: last_days, "displayValue": last_days})
+            search_request.query.filter.add_filter(time_condition)
+
+        if threat_model_names:
+            rule_condition = FilterCondition()\
+                .set_path(alert_attributes.Alert_Rule_Name)\
+                .set_operator("In")
+            for threat_model_name in threat_model_names:
+                rule_condition.add_value({alert_attributes.Alert_Rule_Name: threat_model_name, "displayValue": "New"})
+            search_request.query.filter.add_filter(rule_condition)
+
+        if alertIds:
+            alert_condition = FilterCondition()\
+                .set_path(alert_attributes.Alert_ID)\
+                .set_operator("In")
+            for alertId in alertIds:
+                alert_condition.add_value({alert_attributes.Alert_ID: alertId, "displayValue": "New"})
+            search_request.query.filter.add_filter(alert_condition)
+
+        if device_names:
+            device_condition = FilterCondition()\
+                .set_path(alert_attributes.Alert_Device_HostName)\
+                .set_operator("In")
+            for device_name in device_names:
+                device_condition.add_value({alert_attributes.Alert_Device_HostName: device_name, "displayValue": device_name})
+            search_request.query.filter.add_filter(device_condition)
+
+        if user_names:
+            user_condition = FilterCondition()\
+                .set_path(alert_attributes.Alert_User_Identity_Name)\
+                .set_operator("In")
+            for user_name in user_names:
+                user_condition.add_value({alert_attributes.Alert_User_Identity_Name: user_name, "displayValue": user_name})
+            search_request.query.filter.add_filter(user_condition)
+
+        if alert_statuses:
+            status_condition = FilterCondition()\
+                .set_path(alert_attributes.Alert_Status_ID)\
+                .set_operator("In")
+            for status in alert_statuses:
+                status_id = ALERT_STATUSES[status.lower()]
+                status_condition.add_value({alert_attributes.Alert_Status_ID: status_id, "displayValue": status})
+            search_request.query.filter.add_filter(status_condition)
+
+        if alert_severities:
+            severity_condition = FilterCondition()\
+                .set_path(alert_attributes.Alert_Rule_Severity_ID)\
+                .set_operator("In")
+            for severity in alert_severities:
+                severity_id = ALERT_SEVERITIES[severity.lower()]
+                severity_condition.add_value({alert_attributes.Alert_Rule_Severity_ID: severity_id, "displayValue": severity})
+            search_request.query.filter.add_filter(severity_condition)
+
+        if descending_order == 'True':
```

Comment:
You declare the `descending_order` as bool, but compare it to str.
I think it would be better to use `argToBool` method and use it as a boolean argument

Reply:
Done